### ISSUE TITLE
version 3.9.2 quote attribution

### ISSUE THREAD

**Issue by zeynepalici** [stanfordnlp/CoreNLP#1183]
Hi,

I want to do a quote attribution task same as the Muzny paper:

        Properties props = new Properties();
        props.setProperty("coref.algorithm", "neural");
        props.setProperty("file", "quote-example.txt");
        props.setProperty("outputFormat", "text");
        props.setProperty("ner.useSUTime", "false");
        props.setProperty("charactersPath","characters.txt");

        StanfordCoreNLP pipeline = new StanfordCoreNLP(props);
        CoreDocument document = new CoreDocument(text);
        pipeline.annotate(document);

But Tiffany and Savannah are different characters. How Is it possible even if canonical speaker is Savannah, the orşgşnal speaker is Tiffany.

 Example: quote:
"My file is gone. Someone broke in here and stole that file folder."
Example: original speaker of quote:
Tiffany
Example: canonical speaker of quote:
Savannah

characters.txt
Savannah;None;Savannah
Connor;M;Connor;Connor Wells
Melanie;F;Melanie;Melanie Wedgewood
Niles;M;Niles
Jamie;None;Jamie
Jimmy;M;Jimmy;Jimmy Takata
Tiffany;F;Tiffany
Letty;F;Letty
Brian Donahue;M;Brian Donahue
Denny;None;Denny
Annie;F;Annie
Thomas;M;Thomas

**Comment by AngledLuffa** [comment]
Sorry, but there is basically no chance we try to debug a specific error in
a version from 3 years ago.  Anyway, if you're looking to recreate exact
results, doesn't that include the bugs?  :P

If you can send us a more detailed example, eg something we can compile and
run with no further effort on our part, and the error still happens on the
latest version, we can try to debug it and you can see if the patch applies
to the old version.

**Comment by zeynepalici** [comment]
Of course, thank you for your answer. Exactly I just want to use a character list for quote attribution. This character list includes aliases and the gender of characters. I wonder that whether I am doing wrong in running the pipeline for quote attribution with a character file.

If you have a chance, can you share a java file or code segment for quote attribution using a character file? At this point, my question is independent of the version. Even if you can share it for the current version, it would be very beneficial for me :)

**Comment by AngledLuffa** [comment]
I have literally never touched this aside from debugging NPE (one reason I
have no desire to investigate the 3.9.2 version at all) and the person who
worked on this has long since graduated.  Would you send me a sample of
what you are doing so far and I'll see if I can make it work?

Thanks
